src/app/organization/tellers/tellers.component.html

```diff
@@ -47,14 +47,14 @@
       <ng-container matColumnDef="actions">
         <th mat-header-cell *matHeaderCellDef> Actions </th>
         <td mat-cell *matCellDef="let teller">
```

Comment:
```suggestion
        <td mat-cell *matCellDef="let teller" (click)="$event.stopPropagation()">
```
Good work! But lets just shift it to the parent cell here.